jobs/build.Jenkinsfile

```diff
@@ -137,6 +137,14 @@ lock(resource: "build-${params.STREAM}") {
         cat /cosa/coreos-assembler-git.json
         """)
 
+        // add any additional root CA cert before we do anything that fetches
+        pipeutils.tryWithCredentials([file(credentialsId: 'additional-root-ca-cert', variable: 'ROOT_CA')]) {
+            shwrap('''
+                cp $ROOT_CA /etc/pki/ca-trust/source/anchors/
+                /usr/lib/coreos-assembler/update-ca-trust-unpriv
+            ''')
+        }
```

Comment:
Is this only needed for the `build` job?

Reply:
Good catch, it was needed in the `bump-lockfile` job too! I made it into a helper function. It's tempting to put some of this stuff into coreos-ci-lib. Probably makes sense to do that eventually.

Reply:
I feel like we would need this also in `build-arch.Jenkinsfile`? and possibly need to plumb that through to the "remote-session" in that job?

Maybe we don't need it in that job for some reason that I don't fully understand?

Reply:
I punted on that because I wasn't sure what the plan was there. If all the worker nodes we're planning to use will have access to the internal network, then I can wire through the root CA cert to the remote sessions. But if not, it might not be needed depending on how we'll proxy access.

Right now, I'm just trying to get at least x86_64 working though. Let's get this in and then work on the multi-arch bits in follow-ups?